include/aspect/velocity_boundary_conditions/interface.h

```diff
@@ -59,12 +59,23 @@ namespace aspect
          * Initialization function. This function is called once at the
          * beginning of the program after parse_parameters is run and after
          * the SimulatorAccess (if applicable) is initialized.
+         *
+         * @depricated Use initialize(types::boundary_id boundary_id) instead.
          */
         virtual
         void
         initialize ();
```

Comment:
Thanks! That is a very useful marcro indeed. I just looked at the cookbooks, tests and benchmarks. Non of them seem to use the initialize of the velocity boundary conditions. I found some other things which could be changed in an other pull request, like that compressiblity.cc in the tests uses the deprecated boundary_indicator an the burstede benchmark has a lot of unused parameters.